featuretools/feature_base/features_deserializer.py

```diff
@@ -140,6 +140,8 @@ def _deserialize_feature(self, feature_name):
 
         args = feature_dict["arguments"]
         feature = cls.from_dictionary(args, self.entityset, dependencies, primitive)
+        if "series_library" in feature_dict:
+            feature.primitive.series_library = feature_dict["series_library"]
```

Comment:
This works, but wondering if this is the best place to set series library on the primitivie

Reply:
As in, within `_deserialize_feature`?